frontend/src/scenes/session-recordings/player/list/consoleLogsListLogic.ts

```diff
@@ -145,7 +113,22 @@ export const consoleLogsListLogic = kea<consoleLogsListLogicType>([
                         }
                     })
                 })
-                return logs
+                return filters?.query
```

Comment:
I checked this out locally and noticed this selector is called pretty often. I think calling `new Fuse()` on every frame of the recording is expensive. I think we should pull out the Fuse part to its own minimal selector based on `logs` which should only be recreated when the logs list changes

Reply:
Hmm, I can't seem to reproduce the logs being recreated on every frame of the recording because it looks like it's only recalculated on `sessionPlayerData` being updated, which only happens when a snapshot is fetched. Are you referring to it being recalculated when metadata and snapshots are fetched separately?